highfidelity/hifi
# Teleportation

This PR adds teleportation to the array of movement options.

To test:
1. Download this build and reload your default scripts.  You should see teleport.js in the defaults now.


To teleport: 
1. Press and hold the very middle of the Vive controller thumbpad  and the trigger to activate teleportation.  Or the A/X button on the Oculus Touch controller.  Or the 1 button on the Hydras.
2. Point the target toward where you'd like to go.
3. Release either the thumb or the trigger.
